app/assets/stylesheets/application.css

```diff
@@ -15,16 +15,25 @@
  */
 
 .navbar {
-    background-color: lightgray;
-    height: 50px;
+  background-color: lightgray;
+  height: 50px;
 }
 .navbar_logo {
-    color: black;
+  color: black;
    
 }
 .category {
-    color: black;
+  color: black;
 }
 .subcategory {
-    color: black;
+  color: black;
+}
+
+#js-search {
+  height: 40px;
+  width: 90px;
+}
```

Comment:
add single line after each styling

Reply:
it is not working
because our search method is handling it

Reply:
added